crates/common/src/lib.rs

```diff
@@ -534,6 +535,48 @@ pub fn calculate_class_commitment_leaf_hash(
     )
 }
 
+pub fn deployed_contract_address(
+    constructor_calldata: impl Iterator<Item = CallParam>,
+    contract_address_salt: &ContractAddressSalt,
+    class_hash: &ClassHash,
+) -> ContractAddress {
```

Comment:
Can we make this part of `ContractAddress` instead of a free-standing function? Would make it a lot easier to discover.

Reply:
sure!